src.ts/multisig.ts

```diff
@@ -0,0 +1,306 @@
+                        throw this.signer.provider.checkTransactionReceipt(receipt, errors.CALL_EXCEPTION, "confirm multisig transaction failed", {
+                            method: "auth-signMutiSigTx",
+                            receipt
+                        });
+                    });
+                });
+            });
+        });
+    }
+
+    private sendRawTransaction(transaction: TransactionRequest, overrides?: any) {
+        if (!this.provider) { errors.throwError('missing provider', errors.NOT_INITIALIZED, { argument: 'provider' }); }
+
+        return populateTransaction(transaction, this.provider, this.address).then((tx) => {
+            return this.sign(tx, overrides).then((signedTransaction) => {
+                return this.provider.sendTransaction(signedTransaction, overrides).catch(error => {
+                    // Clear the cached nonce when failure happened to prevent it out of sequence
+                    this.clearNonce();
+                    throw error;
+                });
+            });
+        });
+    }
+
+     /**
+     * Create multisig wallet
+     * @param properties multisig properties
+     * @param signer signer wallet
+     * @param overrides options
+     */
+    static create(properties: MultiSigWalletProperties, signer: Signer, overrides?: any): Promise<TransactionResponse | MultiSigWallet> {
+        if (!Signer.isSigner(signer)) {
+            errors.throwError('create multisig wallet transaction require signer', errors.MISSING_ARGUMENT, { arg: 'signer' });
+        }
+        if (properties && 'string' === typeof (properties)) {
+            properties = JSON.parse(properties);
+        }
+
+        checkProperties(properties, {
+            owner: true,
+            threshold: true,
+            signers: true,
+        }, true);
+
+        return resolveProperties({ signerAddress: signer.getAddress() }).then(({ signerAddress }) => {
+            if (!signerAddress) {
+                errors.throwError('create multisig wallet require signer address', errors.MISSING_ARGUMENT, { required: 'signer' });
+            }
+            properties.owner = signerAddress; // Set signer address as owner
+
+            let multisig: MultiSigWalletProperties = checkFormat({
+                owner: checkString,
+                threshold: checkNumber,
+                signers: checkAny,
+            }, properties);
+
+            let transaction = signer.provider.getTransactionRequest("multisig", "auth-createMultiSigAccount", {
+                from: multisig.owner,
+                threshold: multisig.threshold,
+                signers: multisig.signers
+            });
+            transaction.fee = signer.provider.getTransactionFee(undefined, undefined, { tx: transaction });
+            
+            return signer.sendTransaction(transaction, overrides).then((response) => {
+                
+                let groupAddress = deriveAddress(signerAddress, signer.getNonce());
```

Comment:
The calculation for the groupAddress will be incorrect when the `bulkSend` flag is enable during the sendTransaction, the nonce will be increase by one internally. It should be calculate before sendTransaction.